docs/styles/styles.scss

```
@@ -203,6 +205,18 @@ a:visited {
 
 h2 {
   margin-top: 2.4rem;
+
```

```suggestion
scroll-margin-top: 1rem;
```

Nit: Let's add this just to give our headings some breathing room when scrolled to